src/helpers/currency.js

```diff
@@ -18,3 +19,13 @@ export const getSymbol = (currency, locale = userLocale, amount = 0, isNarrowSym
 
     return string.replace(/\d/g, "").trim();
 };
+
+export const getValueWithCurrency = (amount, currency) => {
+    const userLocale = getUserLocale();
```

Comment:
pass additional param called locale (optional field) and incase locale is there can you use it, we will be using that once we deploy `localisation` story. 

const userLocale = locale ?? getUserLocale();

Reply:
Sure. Done.